setup.py

```diff
@@ -37,7 +37,10 @@ def find_version(*parts):
     author='Jannis Leidel',
     author_email='jannis@leidel.info',
     packages=find_packages(),
-    install_requires=['six'],
+    install_requires=[
+        'six',
+        'futures; python_version == "2.7"',
```

Comment:
I didn't know about this syntax. This is cool, and helped me find this article: https://hynek.me/articles/conditional-python-dependencies/